doc/source/dev/gitwash/development_workflow.rst

```diff
@@ -195,27 +187,78 @@ Standard acronyms to start the commit message with are::
    REL: related to releasing numpy
 
 
+.. _asking-for-merging:
+
+Asking for your changes to be merged with the main repo
+=======================================================
+
+When you feel your work is finished, you can file a pull request and
+submit an optional code review. Code reviews are typically submitted when an
```

Comment:
What does "and submit an optional code review" mean? Maybe ask for a code review on the mailing list?